docs/man/man1/gmoccapy.1

```diff
@@ -27,14 +27,14 @@
 gmoccapy \- TOUCHY LinuxCNC Graphical User Interface
 .SH SYNOPSIS
 .B gmoccapy
-\fI\-ini\fR \fIINIFILE\fR
+\fI\-ini\fR \fI<INI file>\fR
```

Comment:
Still some space for optimization left :D 
I have no idea why this is shown like this:
![grafik](https://user-images.githubusercontent.com/67957916/189542534-15ac492b-5f41-4cab-8a04-5914eb630d56.png)
(also in the other occurrences)